perun-base/src/main/java/cz/metacentrum/perun/core/api/AttributeRights.java

```diff
@@ -19,7 +19,7 @@
 	/**
 	 * Role, that specifies the users, who have the rights upon the attribute.
 	 */
-	private Role role;
+	private String role;
```

Comment:
You have to change the equals method to use the `equals` method.

Reply:
Done